docs/apm/traces/tracing-ingest.md

```diff
@@ -0,0 +1,59 @@
+---
+id: tracing-ingest
+title: Use Tracing to Troubleshoot Ingest
+sidebar_label: Use Tracing to Troubleshoot Ingest
+description: Use tracing to to search the Audit Event Index for all events that impact ingest.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The [Audit Event Index](/docs/manage/security/audit-event-index/) contains event logs in JSON format on account activities, allowing you to monitor and audit changes. You can use the `tracingIngest` source category to search the Audit Event Index for all events that impact [log ingestion](/docs/manage/ingestion-volume/log-ingestion/).
+
+## Example query
+In the following example, events logged for `_index=sumologic_system_events AND _sourceCategory=tracingIngest` indicate some impact on ingest. The example query assumes all event types are events to be informed about.
+```
+_index=sumologic_system_events AND _sourceCategory=tracingIngest
+| json field=_raw "eventName"
+| timeslice 1h
+| count by _timeslice, eventName
+| transpose row _timeslice column eventName
+```
+
+## Ingest events
+The following ingest events and accompanying reason messages appear in tracing ingest searches.
+
+### ServiceMapEdgesDropped
+The service map edge was dropped due to the number of edges is bigger than a limit:<br/>`Dropped ${numberOfDeletedEdges} edges because limit of number of edges is exceeded. (...)` 
```

Comment:
I think there is a grammar error `due to the number of edges is bigger`. Is should be changed to being.

Reply:
Similar note for ServiceMapNodesDropped dropped below.

Reply:
Done.